gatsby-config.js

```diff
@@ -209,10 +209,6 @@ module.exports = {
               maxWidth: 850,
             },
           },
-          autoLinkHeaders,
-          // This MUST come after `gatsby-remark-autolink-headers` to ensure the
-          // link created for the icon has the proper id
-          'gatsby-remark-custom-heading-ids',
```

Comment:
i think we may still want the `gatsby-remark-custom-heading-ids` plugin

Reply:
ok, i can put that back